python/datasources/acs_population.py

```diff
@@ -291,15 +287,22 @@ def write_to_bq(self, dataset, gcs_bucket):
         frames['by_sex_%s' % self.get_geo_name()] = self.get_by_sex(
             frames[self.get_table_name_by_sex_age_race()])
 
+        # Generate national level datasets based on the state datasets
+        if not self.county_level:
+            for demo in ['age', 'race', 'sex']:
+                state_table_name = 'by_race_state_std' if demo == 'race' else 'by_%s_state' % demo
```

Comment:
would it make sense for us to coordinate `race` vs `race_ethnicity` throughout the backend? maybe just use race? It just seems to make for more complicated code that sometimes we name things w/ the word ethnicity attached and sometimes we don't

Reply:
Yeah its definitely an issue throughout the codebase. For now it probably doesnt matter that much but if/when the cdc actually does split out race/ethnicity its something we will need to tackle.